app/views/content/is-teaching-right-for-me/computing/_article.html.erb

```diff
@@ -0,0 +1,90 @@
+<div class="row inset">
+  <section class="col col-720">
+<p>Teaching computing you'll help pupils become responsible and confident users of information and technology.</p>
+<p>In an increasingly digital world, you could inspire pupils to look at a future role in technology. With roles in technology making up 14% of all job opportunities in the UK.</p>
+<p><a href="/funding-and-support/scholarships-and-bursaries">Tax-free bursaries of £28,000 and scholarships of £30,000 are available for eligible trainee computing teachers</a>.</p>
+</section>
+  <section class="col col-space-s col-content-max">
+<%= render TextBlockWithTwoImagesComponent.new(
+heading: "Help your pupils learn new skills",
+colour: "pink",
+image_1_path: "static/images/content/hero-images/0035.jpg",
+image_1_alt: "",
+image_2_path: "static/images/content/hero-images/0037.jpg",
+image_2_alt: "",
+) do %>
+<p>Computing focuses on 3 areas:</p>
+<ul>
+<li>information and communication technology</li>
+<li>digital literacy</li>
+<li>computer science</li>
+</ul>
+<p>You'll inspire pupils to access new fields and teach them new, transferable skills that will open lots of sectors, from engineering to healthcare. Leading them to future careers like software engineering and web design.</p>
```

Comment:
@caitlin-reidy don't think we need to have new twice. new fields and teach them **new**, transferable skills think we could lose the 2nd one? If they're teaching them then I guess that implies it's a new skill as they're learning it?

Reply:
@caitlin-reidy You'll inspire pupils to access new fields - this sounded a bit formal. Perhaps it could be a little more conversational? You'll get pupils excited to learn about new fields - and can we give a couple of examples?

Reply:
@caitlin-reidy not sure this bit of the sentence quite makes sense transferable skills **that will open lots of sectors,** do you mean open up job opportunities in lots of sectors?

Reply:
@caitlin-reidy this isn't a full sentence Leading them to future careers like software engineering and web design. So might be better to split the 1st sentence and make it shorter and combine the end of it with the 2nd sentence? 

You'll get pupils excited to learn about new fields. You'll teach them transferable skills, that will open up opportunities in sectors including engineering and healthcare, and that could lead to careers such as software engineering and web design.